stellar-soroban-rpc/stellar-core_captive-futurenet.cfg

```
@@ -0,0 +1,33 @@
+# This is a sample configuration from the stellar-captive-core package.
+# Please note that only HOME_DOMAINS and VALIDATORS tables are required.
+
+# IMPORTANT
+# You need to change domains and validators below to fit your needs
+
+# captive core config for futurenet
+NETWORK_PASSPHRASE="Test SDF Future Network ; October 2022"
+# disable the web service port, not used
+HTTP_PORT=0
+PUBLIC_HTTP_PORT=false
+# To avoid conflicts with the core instance
+PEER_PORT=11726
+DATABASE="sqlite3:///var/lib/stellar/soroban-rpc/captive-core/stellar-rpc.db"
+
+EXPERIMENTAL_BUCKETLIST_DB=true
+EXPERIMENTAL_BUCKETLIST_DB_INDEX_PAGE_SIZE_EXPONENT=12
+ENABLE_SOROBAN_DIAGNOSTIC_EVENTS=false
+
+FAILURE_SAFETY=0
+UNSAFE_QUORUM=true
+
+[[HOME_DOMAINS]]
+HOME_DOMAIN="futurenet.stellar.org"
+QUALITY="MEDIUM"
+
+[[VALIDATORS]]
+NAME="sdf_futurenet_1"
+HOME_DOMAIN="futurenet.stellar.org"
+PUBLIC_KEY="GBRIF2N52GVN3EXBBICD5F4L5VUFXK6S6VOUCF6T2DWPLOLGWEPPYZTF"
+ADDRESS="core-live-futurenet.stellar.org:11625"
+HISTORY="curl -sf http://history-futurenet.stellar.org/{0} -o {1}"
```

Should we use https here?
```suggestion
HISTORY="curl -sf https://history-futurenet.stellar.org/{0} -o {1}"
```